### PR TITLE
Write cronjob for update round

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
Just add a cronjob to update power round periodically

### CODE REVIEW

File: src/utils/powerBoostingUtils.ts [hunk 1]
```diff
@@ -14,10 +14,8 @@ export const getRoundNumberByDate = (
 
   const now = getTimestampInSeconds(date);
 
-  // use math.ceil because rounds starts from 1 not zero
-  const previousGivbackRound = Math.ceil(
-    (now - firstGivbackRoundTimeStamp) / givbackRoundLength,
-  );
+  const previousGivbackRound =
+    Math.floor((now - firstGivbackRoundTimeStamp) / givbackRoundLength) + 1;
```
Comment (aminlatifi) [inline src/utils/powerBoostingUtils.ts:18]: Let's change its name to round or givbackRound! it's not previousGivbackRound anymore

File: src/services/cronJobs/updatePowerRoundJob.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+export const runUpdatePowerRoundCronJob = () => {
+  logger.debug(
+    'runUpdatePowerRoundCronJob() has been called, cronJobTime',
+    cronJobTime,
+  );
+  schedule(cronJobTime, async () => {
+    const roundNumber = getRoundNumberByDate(new Date()).previousGivbackRound;
+    logger.debug('runUpdatePowerRoundCronJob roundNumber', roundNumber);
+    await setPowerRound(roundNumber);
```
Comment (aminlatifi) [inline src/services/cronJobs/updatePowerRoundJob.ts:21]: We must refresh two views

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/utils/powerBoostingUtils.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply src/utils/powerBoostingUtils.ts:18]: Done

File: src/services/cronJobs/updatePowerRoundJob.ts (hunk 2 above)
Comment (mohammadranjbarz) [reply src/services/cronJobs/updatePowerRoundJob.ts:21]: Done

**Review by aminlatifi — APPROVED** [review]
LGTM. Thanks @mohammadranjbarz

**Pull request merged**